### ISSUE TITLE
start up failed with influxdb-spring 3.1.0 

### ISSUE THREAD

**Issue by yikuo123** [influxdata/influxdb-client-java#250]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. add `implementation 'com.influxdb:influxdb-spring:3.1.0'` 
2. compile and start up my spring boot application

__Expected behavior:__

just works

__Actual behavior:__

```
java.lang.IllegalStateException: Error processing condition on com.influxdb.spring.health.InfluxDB2HealthIndicatorAutoConfiguration

Caused by: java.lang.IllegalStateException: Failed to introspect Class [com.influxdb.spring.influx.InfluxDB2AutoConfiguration] from ClassLoader [jdk.internal.loader.ClassLoaders$AppClassLoader@1f89ab83]

Caused by: java.lang.NoClassDefFoundError: com/influxdb/client/reactive/InfluxDBClientReactive

Caused by: java.lang.ClassNotFoundException: com.influxdb.client.reactive.InfluxDBClientReactive
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581) ~[na:na]
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178) ~[na:na]

```

__Specifications:__
 - Client Version: 3.1.0
 - InfluxDB Version: 2.0.6
 - JDK Version: 11
 - Platform: Windows

**Comment by bednar** [comment]
Hi @yikuo123,

thanks for using our client. 

How looks like your `application.properties`?

Regards

**Comment by yikuo123** [comment]
> Hi @yikuo123,
> 
> thanks for using our client.
> 
> How looks like your `application.properties`?
> 
> Regards

it’s in yaml format and something like this  :

``` yml
influx:
  url: http://localhost:8086
  org: xxx
  bucket: demo
  token: HqBlE-pRpU4wwQR9zhOP_JMNSvXn7OsdkBbAOqUh4sVGOLYbPcAcvhXsgRiO_MyfDF99Fa2qmg7F_DGu6H9BkQ==
  log-level: body
```

**Comment by ccw55ccw** [comment]
ihave same issue,If I don't dependency influxdb-client-reactive will be an error, because com/influxdb/spring/influx/InfluxDB2AutoConfiguration.java:80 quoted InfluxDBClientReactive.class,so i must denpendency  influxdb-client-reactive

**Comment by bednar** [comment]
@ccw55ccw, @yikuo123 we've prepared PR https://github.com/influxdata/influxdb-client-java/pull/252 with the fixed version. The develop version will be accessible after accept the PR, the fixed stable version in regular release of client.

**Comment by bednar** [comment]
@ccw55ccw, @yikuo123 

the issue is fixed in [3.2.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/26). You could use a preview version by:

```xml
<dependency>
  <groupId>com.influxdb</groupId>
  <artifactId>influxdb-spring</artifactId>
  <version>3.2.0-SNAPSHOT</version>
</dependency>
```

Regards
